bosh/opsfiles/production.yml

```diff
@@ -0,0 +1,68 @@
+- type: replace
```

Comment:
WDYT about making these variables instead of operations?

Reply:
I don't think it matters either way, but if there's a convention I should be following, I'm happy to follow it.  I did it this way because it was the least-effort way to de-spruce things.

Reply:
Convention here calls for required property value substitution to be variables, larger wide sweeping block changes/redactions/optional components to be operators.